packages/framework/fluid-static/src/rootDataObject.ts

```diff
@@ -21,10 +21,19 @@ import {
 } from "./types";
 import { isDataObjectClass, isSharedObjectClass, parseDataObjectsFromSharedObjects } from "./utils";
 
+/**
+ * TODO
+ */
 export interface RootDataObjectProps {
+    /**
+     * TODO
+     */
     initialObjects: LoadableObjectClassRecord;
 }
 
+/**
+ * TODO
+ */
```

Comment:
```suggestion
/**
 * The entry-point/root collaborative object of the fluid container for both the tinylicious-client and the azure-client. This class abstracts the dynamic code required to build a fluid container into a static representation for our end customers.
 */
```

Reply:
Thanks! To avoid abstraction leak, I omitted the tinylicious / azure note, though.